array-list/myArrayList.java

```diff
@@ -33,6 +33,35 @@ public boolean contains(E elm) {
   private void expand() {
     arr = Arrays.copyOf(arr, arr.length * 2);
   }
+  
+  private int indexOf(E elm){
+    for(int i = 0; i < size; i++){
+      if(elm.equals(arr[i])){
+        return i;
+      }
+    }
+    return -1;
+  }
+
+  private E remove(E elm){
+    
+  int len = arr.indexOf(elm);
+  if(len == -1){
+  return;
+  } else{
```

Comment:
you will return when len == -1, so, you don't need to write else, just write the code. The else keyword is not important here